lib/schemas/config.js

```diff
@@ -1,17 +1,14 @@
 const yup = require('yup');
 
+// Add URL once this is resolved: https://github.com/jquense/yup/issues/800
 const configSchema = yup.object().shape({
-  apiServerUrl: yup
-    .string()
-    .url()
-    .nullable(),
+  apiServerUrl: yup.string().nullable(),
```

Comment:
Comment: Rather than removing the URL validation and requiring just a string or null, I would prefer to allow `localhost` or a valid URL. A custom regex pattern could be one solution.

Reply:
Do you mean implement the validation locally instead of using Yup? My initial thought was to submit a PR to fix the validation in Yup. But there is already an open PR for that. Then I looked into extending yup validation to allow for what we want but I couldn't completely figure that out. But yes, that would be a solution. The reason I chose to just remove it till they fix it on Yup is because currently only internal projects use the node SDK so we can control what goes and the validation is not that vital. Let me know what you think.